tests/model_registry/rbac/conftest.py

```diff
@@ -89,6 +92,51 @@ def sa_token(service_account: ServiceAccount) -> str:
         raise
 
 
+@pytest.fixture(scope="function")
+def new_group(
+    request: pytest.FixtureRequest, test_idp_user_session: TestUserSession, admin_client: DynamicClient
```

Comment:
Whenever we use admin_client, we should use it before other session scoped fixtures. The reason being, it is used always before other session scoped fixtures.